python_on_whales/components/container/cli_wrapper.py

```diff
@@ -387,14 +389,20 @@ def remove(self, force: bool = False, volumes: bool = False) -> None:
         )
 
     def start(
-        self, attach: bool = False, interactive: bool = False, stream: bool = False
+        self,
+        attach: bool = False,
+        detach_keys: Optional[str] = None,
+        interactive: bool = False,
+        stream: bool = False,
```

Comment:
```suggestion
        interactive: bool = False,
        stream: bool = False,
        detach_keys: Optional[str] = None,
```
To avoid breaking backward compatibility. We'll disally positional arguments soon anyway for this method, and the order won't matter then. But for now, we don't want any silent breakage.

Reply:
Makes sense, I'll make the relevant changes!